docs/core/linux-prerequisites.md

```diff
@@ -91,9 +106,10 @@ The installer bash script is used in automation scenarios and non-admin installa
 ## Install .NET Core dependencies for Red Hat Enterprise Linux (RHEL) 7 Server
 
 > [!Warning]
```

Comment:
- I've tried everything I can think of to align the code samples inside the tabs. I will follow up with Rob about it and see what can be done.
- Yes, agreed. Moving the warning to first step is a good idea.